package.json

```diff
@@ -49,7 +49,7 @@
     "rename-package": "node -r ./scripts/ts-node-register ./scripts/rename-package.ts",
     "run:published": "node ./scripts/monorepo/runPublished.js",
     "runto:lerna": "node ./scripts/monorepo/runTo.js",
-    "satisfied": "satisfied --skip-invalid --ignore \"sass\"",
+    "satisfied": "satisfied --skip-invalid --ignore \"sass|@testing-library\"",
```

Comment:
without this we run into an error 
![image](https://user-images.githubusercontent.com/1223799/110836740-b18b3280-82a0-11eb-9289-5e289c725448.png)

↓

![image](https://user-images.githubusercontent.com/1223799/110836964-fd3ddc00-82a0-11eb-8de5-1539499f6d28.png)


I'd consider to get rid of those checks once we migrate everything to single version policy